README.md

```diff
@@ -82,3 +82,4 @@ Maintainers
 
 * Christophe Coevoet [stof](https://github.com/stof)
 * Pete Otaqui [pete-otaqui](http://github.com/pete-otaqui)
+* Alex [aik099](https://github.com/aik099)
```

Comment:
Please change `Alex` to `Alexander Obuhovich` please.

---

To be consistent with full name approach used for other maintainers.

Reply:
Sorry, I haven't found your full name, checked GitHub account